modules/next_stage/README.md

```diff
@@ -0,0 +1,51 @@
+# Next Stage
+
+## Purpose
+
+The `next_stage` module provides a method of start the "next stage"
+of a candidate session. The "Stage" of an session goes from Screening
+to Visit to Approval, and the Screening and Visit stages may have
+different instruments administered. The `next_stage` module prompts
+for the date of the visit and uses the data it gathers to populate
+the test battery for the Visit stage.
+
+However, note that the Screening stage is mostly historic and poorly
+tested.  Most LORIS studies prefer to have a separate "Screening"
+visit, rather than a screening stage for each visit. The result is
+that the `next_stage` module is mostly used simply to populate the
+test battery for a session.
+
+## Intended Users
+
+The `next_stage` module is used by data entry staff in order to
+start the Visit stage of an instrument.
+
+## Scope
+
+The `next_stage` module only moves from the Screening to Visit stage
+for a session. The final Approval or Recycling Bin stage is reached
+by sending to DCC (or Recycling Bin) on the `instrument_list` page.
+
+## Permissions
+
+Both the `data_entry` permission is required, and the user must be
```

Comment:
Is there 2 permissions required?
one suggestion is : "The `data_entry` permission is required. The user must also be at the same CenterID ..."

or "... The user wil only see the timepoint for CenterID where he is activated" or something similar

Reply:
Yes, both the explicit permission with the permission code, and the same CenterID (which isn't really a permission in the code sense, but affects access) is required.

I updated it to your wording, I think it's more clear.